lib/export/src/Export.hs

```diff
@@ -304,12 +353,12 @@ auxppTerm ppLit t = (ppTerm t, getHdTerm t)
         . map ppTerm
         $ ts
     ppFun f ts =
-      text (BC.unpack f ++ "(") <> fsep (punctuate comma (map ppTerm ts)) <> text ")"
+      text (ppFunSym f ++ "(") <> fsep (punctuate comma (map ppTerm ts)) <> text ")"
     getHdTerm tm = case viewTerm tm of
       Lit (Con (Name PubName n)) ->
         if List.elem (show n) ["g", "one", "zero"]
           then S.empty
-          else S.singleton (Sym "free" (show n) ":bitstring" [])
+          else S.singleton (Sym "free" ("s" ++ show n) ":bitstring" [])
```

Comment:
The 's' is just prepended here instead of using `sanitizeSymbol`, because that function only does the prepending for reserved keywords and symbols starting with a digit. For free bitstrings however, we ALWAYS want the leading 's', to also avoid clashes with function names, rule names, event names etc. We could also do it like that for variables and function names (where we use sanitizeSymbol now), but I thought if we did it in all other places it might not be needed there, and I thought it would be better to leave as much as possible of the original naming as it is.

(I also added the above as a comment in the file, feel free to edit if it does not fit there)